assets/js/blocks/product-collection/inspector-controls/author-control.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEntityRecords } from '@wordpress/core-data';
+import {
+	FormTokenField,
+	// @ts-expect-error Using experimental features
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ProductCollectionQuery } from '../types';
+
+interface Author {
+	id: string;
+	name: string;
+}
+
+interface AuthorsMapping {
+	mapById: Map< number, Author >;
+	mapByName: Map< string, Author >;
+	names: string[];
+}
```

Comment:
This interface is used only to be extended by `AuthorsInfo`. In that case, I think it could be moved to the `AuthorsInfo` definition without extending unless there's some reason behind that split?

Reply:
I appreciate your input. Upon further review, I realized that I was utilizing `AuthorsMapping` in two different areas. However, due to some recent code changes, I am no longer using it elsewhere. Thank you for bringing this to my attention. I will merge it into `AuthorsInfo`. 🙌🏻